bridge/client/app/evaluation-board/evaluation-board.component.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import {EvaluationBoardComponent} from './evaluation-board.component';
+import {AppModule} from '../app.module';
+import {HttpClientTestingModule} from "@angular/common/http/testing";
+
+describe('ProjectBoardComponent', () => {
+  let component: EvaluationBoardComponent;
+  let fixture: ComponentFixture<EvaluationBoardComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+
+      declarations: [],
+      imports: [
+        AppModule,
+        HttpClientTestingModule,
+      ],
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(EvaluationBoardComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create an instance', () => {
```

Comment:
We could speed up our pipeline by a few minutes by removing these default tests, you might want to remove them too.

Reply:
I have no idea if there is value to those auto-generated "it should create an instance" tests. For this PR I would say it's fine to keep it as it is, but could we discuss this in https://github.com/keptn/keptn/issues/2000?